frontend/packages/pipelines-plugin/src/utils/pipeline-actions.tsx

```diff
@@ -192,7 +196,8 @@ export const stopPipelineRun: KebabAction = (kind: K8sKind, pipelineRun: Pipelin
           {
             op: 'replace',
             path: `/spec/status`,
-            value: 'PipelineRunCancelled',
+            value:
+              version >= '1.9' ? DataState.STOPPED_RUN_FINALLY : DataState.PIPELINE_RUN_CANCELLED,
```

Comment:
we should not compare the version as the version can be in this format 1.10.x which is not a valid number as per the javascript and javascript will throw `Uncaught SyntaxError: Unexpected number` error. So, better to use minor and major return by `usePipelineOperatorVersion` hook. 
Something like this

```suggestion
              version.major === 1 && version.minor < 9
                ? DataState.PIPELINE_RUN_CANCELLED
                : DataState.STOPPED_RUN_FINALLY,
```

Reply:
Updated PR with the said changes